arrocke/gloss-translation
# feat: update profile

## What has changed



* Added ability to update the email, name, and password for a user
* Changing a password or email sends a notification email
* Changing an email doesn't go into affect until it is verified by clicking a link in the email

## Connected Issues



closes #71

## QA Steps



1. Login as a user
2. Change your name
3. Check that the name changed in the header
4. Change your password
5. Logout and try to log in with the new password
6. Check that an email was sent
7. Change your email address
8. Go to the verification URL in the email
9. Check the profile for the new email address
10. Check for an email sent to the old email address
